sys/dev/efidev/efidev.c

```diff
@@ -183,6 +260,44 @@ efidev_ioctl(struct cdev *dev __unused, u_long cmd, caddr_t addr,
 		break;
 	}
 
+#ifdef COMPAT_FREEBSD64
+	cmd = orig_cmd;
+	addr = orig_addr;
+	switch (cmd) {
```

Comment:
Perhaps leave `cmd` and `addr` as-is and just use `switch (orig_cmd)`?  (I think switching on orig_cmd is what I did in ifioctl().)

Reply:
Hm, I did it like this in order to make it as close to looking like the COMPAT_FREEBSD64 code wrapped the inner code